### PR TITLE
Community section rewamp

### LINKED ISSUE #351: Feature Request: revamp "Know the community" section
### Describe the solution you'd like.

We are changing the current "Know the community" section of our website.
Please refer to the below image for the reference.

MacBook Pro 14_ - 2

If are interested to work on this, we might additionally want you to attend our community meetings.
Its recommended that you accomplish this in multiple commits, and open a PR right after the first one.

### PR DESCRIPTION
**Pull request by samyakshah3008**
## Description

I have rewamped the community section of moja global's community website under guidance by @padmajabhol and @HarshCasper  

Style guide followed: https://www.figma.com/file/cpR8K7fBhznhMB70Ym3Ui3/community-website?node-id=0%3A1&t=6kMORArERZpSX00n-1

Dependency add: Better react carousel 
Dependency doc link: https://reactjsexample.com/react-responsive-carousel-component-with-grid-layout-to-easily-create-a-carousel-like-photo-gallery/

Fixes #351 

## Type of change

- New feature (non-breaking change which adds functionality).

## How Has This Been Tested?

This was CSS feature, so I have checked on various mobile and laptop screens. 

## Additional Context (Please include any Screenshots/gifs if relevant)
[moja-global-poc.webm](https://user-images.githubusercontent.com/86982322/211653742-ecd81f56-5bb8-422d-8e18-3c82e560e218.webm)

...

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Hi @samyakshah3008 — This is solid work! Thanks for working on this 💪 

Some of my recommendations to improve this include:

- Improve padding on both ends of the screen. Right now, they seem uneven 👀 
- Can we use arrows or pointers to point to our community users about what an optimized flow would look like? Can we accomodate something similar with our focus on responsiveness? 
- The color schema currently looks pretty dull to me. Can we do something to improve this? 
- How feasible would it be to remove `Engage with community` from the community website? I don't see it fitting any purpose any longer.

@Namyalg @padmajabhol Can you please have a look?

**Review by padmajabhol — CHANGES_REQUESTED** [review]
Screenshot 2023-01-17 at 9 45 36 PM

Bit of a spacing issue here, can you leave uniform padding for both left and right side of the component?

The section somehow covers the entire screen width, which is very different from the figma prototype. 
And in place of the sliders, we had two triangles, can that be done too?

And please make sure that the PR is directed to [community-website-v2](https://github.com/moja-global/community-website/tree/community-website.v2).

File: website/src/components/CommunitySlide/CommunitySlide.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.communityCard {
+    width: 390px;
```
Comment (sohamsshah) [inline website/src/components/CommunitySlide/CommunitySlide.css:2]: prefer `rem` over `px` in css. Border radius values with `px` are okay.

File: website/src/components/CommunitySlide/CommunitySlide.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    height: 259px;
+    border-radius: 15px;
+    background-color: var(--primary-color);
+    margin-bottom: 4rem;
+}
+
+.positionDown {
+
```
Comment (sohamsshah) [inline website/src/components/CommunitySlide/CommunitySlide.css:10]: remove extra line

File: website/src/components/CommunitySlide/CommunitySlide.module.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    color: var(--white-heading-color);
+    line-height: 30px;
+    font-family: 'Poppins', sans-serif;
+    margin-top: 1rem;
+    margin-bottom: 0.3rem;
+}
+
+.cardBodyPills {
+    font-family: 'Poppins', sans-serif;
+    color: var(--secondary-text-color) !important;
```
Comment (sohamsshah) [inline website/src/components/CommunitySlide/CommunitySlide.module.css:57]: remove explicit `!important` styling

File: website/src/components/CommunitySlide/CommunitySlide.module.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    font-size: 20px;
+    color: var(--white-heading-color);
+    line-height: 30px;
+    font-family: 'Poppins', sans-serif;
+    margin-top: 1rem;
+    margin-bottom: 0.3rem;
+}
+
+.cardBodyPills {
+    font-family: 'Poppins', sans-serif;
```
Comment (sohamsshah) [inline website/src/components/CommunitySlide/CommunitySlide.module.css:56]: Why do we want to specify `fontFamily` explicitly?

Font family of the website should be declared globally.

File: website/src/components/CommunitySlide/index.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        <div className={styles.ellipseContainer}>
+          <div className={styles.ellipse}></div>
+          <div className={styles.ellipse}></div>
+          <div className={styles.ellipse}></div>
+        </div>
+        <div className={styles.borderLine}></div>
+        <div className={styles.communityCardBody}>
+          <div className={styles.cardBodyHeader}> {title} </div>
+          {description.map((item) => (
+            <a href={item.link} target="_blank" className={styles.cardBodyPills}>
```
Comment (sohamsshah) [inline website/src/components/CommunitySlide/index.js:19]: please add `key` to the mapped React component.

File: website/src/components/CommunitySlideCarouselSection/index.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Carousel from 'better-react-carousel';
+import features from '../../../websiteData/features';
+import CommunitySlide from '../CommunitySlide';
+
+const CommunitySlideCarouselSection = () => {
+  return (
+    <>
+      <Carousel
+        responsiveLayout={[
```
Comment (sohamsshah) [inline website/src/components/CommunitySlideCarouselSection/index.js:10]: Create a constant array containing details about responsive layout for Carousel and pass that variable as a prop here.

File: website/src/components/CommunitySlideCarouselSection/index.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        cols={3}
+        rows={1}
+        gap={10}
+        loop
+      >
+        {features.map(({ title, description }, index) => (
+          <Carousel.Item>
+            <CommunitySlide
+              title={title}
+              position={(index - 4) % 3 === 0 ? 'down' : 'up'}
```
Comment (sohamsshah) [inline website/src/components/CommunitySlideCarouselSection/index.js:43]: create a function to compute the position. Inline logic inside the `view` is not readable and hard to interpret.

File: website/package.json [hunk 8]
```diff
@@ -25,6 +25,7 @@
     "@emotion/styled": "^11.3.0",
     "@mdx-js/react": "^1.6.21",
     "@svgr/webpack": "^5.5.0",
+    "better-react-carousel": "^1.1.1",
```
Comment (sohamsshah) [inline website/package.json:28]: Any specific reason to use this package? Were there any alternatives you got chance to try upon? Reason for asking is that this package is:
- not well maintained
- 62kbs just for a carousel lib seems too much

Though, not a blocker; just something to think if we can check other alternatives and integrate for better flexibility and reusability.

Cc: @HarshCasper

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Appreciable work @samyakshah3008 ✌🏻 

Have provided some comments, please address them. Let me know if you need any help resolving them!

File: website/src/components/CommunitySlide/CommunitySlide.css (hunk 1 above)
Comment (samyakshah3008) [reply website/src/components/CommunitySlide/CommunitySlide.css:2]: Done ✅

File: website/src/components/CommunitySlide/CommunitySlide.module.css (hunk 4 above)
Comment (samyakshah3008) [reply website/src/components/CommunitySlide/CommunitySlide.module.css:56]: Removed

**Comment by samyakshah3008** [comment]
image

## Here we go with latest changes: 
Fix: Alignment issues 
Fix: previously it was covering 100% width on laptop screens, now changed it to 90% according to figma prototype
Refactor: px to rem converted to maintain code consistency + white spaces removed 

Please let me know the follow-up if any further changes are required.

**Comment by padmajabhol** [comment]
> image
> 
> ## Here we go with latest changes:
> Fix: Alignment issues Fix: previously it was covering 100% width on laptop screens, now changed it to 90% according to figma prototype Refactor: px to rem converted to maintain code consistency + white spaces removed
> 
> Please let me know the follow-up if any further changes are required.

Can you resolve the merge conflicts?

**Pull request closed without merging**